### PR TITLE
Update terms of use text

### PR DESCRIPTION
**Pull request by leomendoza123**
https://trello.com/c/nivDnc2z/6908-qa-register-fix-error-message-for-emails-with-deactivated-and-unclaimed-accounts

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[QA] register - "terms of use" vs "terms and conditions"](https://trello.com/c/geBwW7UN/6905-qa-register-terms-of-use-vs-terms-and-conditions)

**Comment by leomendoza123** [comment]
[[QA] register - correct text in step 3](https://trello.com/c/jUv3TjGu/6906-qa-register-correct-text-in-step-3)

File: src/app/register/components/form-terms/form-terms.component.html [hunk 1]
```diff
@@ -33,6 +33,6 @@ <h3 class="mat-subheading-2" i18n="@@register.termsOfUse">Terms of Use</h3>
   <mat-error
     *ngIf="form.hasError('required', 'termsOfUse') && errorState"
     i18n="@@register.termsRequired"
-    >You must accept the terms and conditions</mat-error
-  >
+    >You must accept the terms of use</mat-error
+  >npm
```
Comment (DanielPalafox) [inline src/app/register/components/form-terms/form-terms.component.html:37]: This npm shouldn't be here

File: src/locale/properties/register/register.en.properties [hunk 2]
```diff
@@ -38,19 +38,18 @@ register.additionalEmails=Additional email
 register.additionalEmailsCanBeDuplicated=Additional email cannot match primary email
 register.additionalEmailAdd=Add another email
 register.additionalEmailInfo=Ensure that you never lose access to your ORCID record by registering additional email addresses on your account.
-register.termsRequired=You must accept the terms and conditions
+register.termsRequired=You must accept the terms of use
```
Comment (DanielPalafox) [inline src/locale/properties/register/register.en.properties:41]: Fix property in the Html is called `register.terms`

File: src/locale/properties/register/register.xx.properties [hunk 3]
```diff
@@ -23,7 +23,6 @@ register.additionalEmailInfo=X
 register.consentTo=X
 register.privacyPolicy=X
 register.and=X
-register.termsAndConditions=X
```
Comment (DanielPalafox) [inline src/locale/properties/register/register.xx.properties:26]: Property `register.terms` is missing in all testing languages

Comment (leomendoza123) [reply src/locale/properties/register/register.xx.properties:26]: that one was removed, now `register.terms` is been use instead

File: src/locale/properties/register/register.en.properties (hunk 2 above)
Comment (leomendoza123) [reply src/locale/properties/register/register.en.properties:41]: This is is a different one an is only use one `form-terms.component.html`

File: src/app/register/components/form-terms/form-terms.component.html (hunk 1 above)
Comment (leomendoza123) [reply src/app/register/components/form-terms/form-terms.component.html:37]: lol

File: src/locale/properties/register/register.en.properties (hunk 2 above)
Comment (leomendoza123) [reply src/locale/properties/register/register.en.properties:41]: register.terms exist for all testing languages

**Pull request merged**